libp2p/transports/tcptransport.nim

```diff
@@ -129,7 +131,8 @@ proc connHandler*(self: TcpTransport,
 proc new*(
   T: typedesc[TcpTransport],
   flags: set[ServerFlags] = {},
-  upgrade: Upgrade): T {.public.} =
+  upgrade: Upgrade,
+  connectionsTimeout = 10.minutes): T {.public.} =
```

Comment:
Is this the same as the previous timeout? If not, we are changing the current behavior, right?

Reply:
It is the same